CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing to The Show Case App
 
-We would love for you to contribute to the project and help make it even better! You can contribute either through code or helping us out on different issues, suggesting features etc. As a contributor here are the guidelines we would like you to follow:
+We would love for you to contribute to the project and help make it even better! You can contribute either through code or helping us out on different issues, suggesting features etc. As a contributor here is the guidelines we would like you to follow:
```

Comment:
The previous version was the correct one. Don't judge yourself that much. :)

Reply:
This commit is done by Grammarly not me! 😅

Reply:
Grammarly makes commits? Whoa. I need that integration. :)

Reply:
That was a figure of speech not literal statement 😛

Reply:
I was joking, sir. :P